src/Sulu/Bundle/WebsiteBundle/Resources/translations/sulu/backend.en.xlf

```diff
@@ -70,22 +70,34 @@
                 <source>website.webspace.settings.key</source>
                 <target>Key</target>
             </trans-unit>
-            <trans-unit id="11" resname="website.webspace.settings.script">
-                <source>website.webspace.settings.script</source>
-                <target>Script</target>
-            </trans-unit>
-            <trans-unit id="12" resname="website.webspace.settings.url">
+            <trans-unit id="11" resname="website.webspace.settings.url">
                 <source>website.webspace.settings.url</source>
                 <target>URL</target>
             </trans-unit>
-            <trans-unit id="13" resname="website.webspace.settings.site-id">
+            <trans-unit id="12" resname="website.webspace.settings.site-id">
                 <source>website.webspace.settings.site-id</source>
                 <target>Site-ID</target>
             </trans-unit>
-            <trans-unit id="14" resname="website.webspace.settings.type.google_tag_manager">
+            <trans-unit id="13" resname="website.webspace.settings.type.google_tag_manager">
                 <source>website.webspace.settings.type.google_tag_manager</source>
                 <target>Google Tag Manager</target>
             </trans-unit>
+            <trans-unit id="14" resname="website.webspace.settings.scriptHeadOpen">
+                <source>website.webspace.settings.scriptHeadOpen</source>
```

Comment:
dont forget nl and fr

Reply:
Maybe you can add this for the nl translation:

```
Script (Head opening) => Script (Begin van de Head)
Script (Head close) => Script (Einde van de Head)
Script (Head opening) => Script (Begin van de Body)
Script (Head close) => Script (Einde van de Body)
```